src/spec/index/publisher.spec.js

```diff
@@ -119,6 +121,11 @@ describe('index/publisher.js', () => {
 					message: {
 						first: 'First',
 						last: 'Last'
+					},
+					context: {
+						user: {
+							username: 'test@test.com'
```

Comment:
`test.com` is a real business domain, better to use `example.com`
But I appreciate test.com has already been used all over the codebase already, and when in rome...